CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+2.0.0
+-----
+Breaking changes:
+* [#49](https://github.com/4finance/uptodate-gradle-plugin/issues/49) Migrating to the new *io.fourfinanceit* packages
+* plugin available at http://plugins.gradle.org - see http://plugins.gradle.org/plugin/io.fourfinanceit.uptodate
```

Comment:
After a merge of this PR or even with https://github.com/4finance/uptodate-gradle-plugin/pull/26 it will not be automatically true. The new plugin from Gradle itself has to be used which is [not the same](https://plugins.gradle.org/docs/submit) as publishing to jcenter (there is a separate repo for plugins). micro-common-release would have to be adjusted to do it - feel free to raise a ticket :).